packages/vscode-extension/lib/wrapper.js

```diff
@@ -173,12 +173,43 @@ export function AppWrapper({ children, initialProps, ..._rest }) {
     [openPreview, closePreview, requestNavigationChange]
   );
 
+  const getInspectorDataForInstance = (node) => {
```

Comment:
I agree, I applied all your suggestions and moved this function along with the logic of traversing tree up to new file `lib/inspector.js`, so that the logic in `wrapper.js` is minimal